app/containers/UserSmsLinkPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useLocation } from 'react-router-dom';
+
+import NotFoundPage from 'containers/NotFoundPage';
+import Player from 'components/Player';
+
+interface LinkDataType {
+  link_type: string;
+  redirect_url: string;
+}
+
+const UserSmsLinkPage = () => {
+  const location = useLocation();
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const { link_type, redirect_url } = location.state as LinkDataType;
+
+  switch (link_type) {
+    case 'website':
```

Comment:
Let's rener here some loader and move redirection to useLayoutEffect - window.location.replace is asynchronous action and causes empty return error :(